modules/migration-configuring-mcg.adoc

```diff
@@ -62,10 +67,11 @@ spec:
      memory: 1Gi
  coreResources:
    requests:
-     cpu: 0.5 <1>
+     cpu: 0.5 <2>
      memory: 1Gi
 ----
 <1> For a very small cluster, you can change the `cpu` value to `0.1`.
+<2> For a very small cluster, you can change the `cpu` value to `0.1`.
```

Comment:
Since this is the same callout sentence, you could actually just use <1> for that second instance above and have them both reference the same callout.

Reply:
I did that in the past and someone told me not to.

Reply:
Hm, that's news to me. As far as I'm aware there aren't any issues with doing that, and I've seen others do it. Let me know if you remember who you heard that from, or we could bring it up to the team and add it to the guidelines if needed.